CODEOWNERS

```
@@ -101,6 +101,7 @@
 /doc/nrf/drivers/suit_flash_ipuc.rst      @nrfconnect/ncs-charon-doc
 /doc/nrf/drivers/images/flash_ipuc/*      @nrfconnect/ncs-charon-doc
 /doc/nrf/drivers/hw_cc3xx.rst             @nrfconnect/ncs-aegir-doc
+/doc/nrf/drivers/mspi_sqspi.rst           @nrfconnect/ncs-doc-leads
```

We will need a new doc group here but for now let's leave my username
```suggestion
/doc/nrf/drivers/mspi_sqspi.rst           @annwoj
```